src/modules/emote_menu/stores/twitch-emotes.js

```diff
@@ -105,7 +119,17 @@ export async function loadTwitchEmotes() {
   const isDark = settings.get(SettingIds.DARKENED_MODE);
   const tempSets = {};
 
-  for (const {owner, id, emotes} of data.channel.self.availableEmoteSets) {
+  const {user} = data;
+
+  const subscriptionSets = user.subscriptionProducts
+    .filter(({id}) => id != null)
+    .map(({id, emotes}) => ({
+      id,
+      owner: user,
+      emotes: emotes.map((emote) => ({...emote, locked: true})),
```

Comment:
we need to figure out a way to dynamically render locks when rendering. as-is a user subscribing or following a channel would not show the locked emotes followed by the then unlocked emotes when they subscribe or follow.